### ISSUE TITLE
Screensaver for specific user

### ISSUE THREAD

**Issue by derekrliang** [derekrliang/backdrop#4]
Would be a nice feature to show only photos of a specific user in a screensaver.

**Comment by Nitish18** [comment]
Hi @derekrliang ,
can I work on this issue ??

**Comment by derekrliang** [comment]
Yes, but might be easier once we get the general screensaver up first.
